logstash-core-plugin-api/lib/logstash-core-plugin-api/version.rb

```diff
@@ -0,0 +1,2 @@
+# encoding: utf-8
+LOGSTASH_CORE_PLUGIN_API = "1.0.0"
```

Comment:
Can we add a comment on what the versioning scheme here is? And also comment that we should not be editing this manually, and rather use the rake script